### ISSUE TITLE
Replace with prefab button

### ISSUE THREAD

**Issue by yuberee** [Facepunch/sbox-issues#6646]
### What can't you do?

Near the "Convert to prefab" I'd like the button to "Replace with prefab", which will remove the object and place a prefab with the exact transform.
I have about 200 light fixtures that aren't prefabs that I need to replace with a unified prefab, and probably more objects as well.
I have done something similar in the past for In This House and it's a long process.

### How would you like it to work?

![Image](https://github.com/user-attachments/assets/62ca2a6c-94cb-4b32-89ab-5794b5b2e995)

A button near the "Convert to prefab" to replace with a prefab instead (On top you can see the prefab I'm using)

### What have you tried?

Manually placing the prefab, copying the transform, applying it, and removing the old object.

**Comment by chrisspieler** [comment]
Agree. It's a common experience that I copy a bunch of objects and then only afterward realize that I should've made them a prefab. This would save the hassle of dragging in prefabs and setting transforms.

**Comment by yuberee** [comment]
UPDATE:

![Image](https://github.com/user-attachments/assets/c9b1d786-36d6-4a0a-99c5-6f4bcef6ff88)

![Image](https://github.com/user-attachments/assets/1a45e90b-d3d3-4ae5-8efd-d6be6b9e0b85)

I made my own solution
